docs/syntax/links.md

```diff
@@ -1,105 +1,139 @@
 # Links
 
-A link contains link text (the visible text) and a link destination (the URI that is the link destination). The link text can include inline elements.
-
-## Inline link
+A markdown link looks like this:
 
 ```markdown
-[Link title](links.md)
+[Link text](destination.md)
 ```
+It has two components:
+- Link **text** enclosed in square brackets `[ ]`
+- Link **destination** enclosed in parentheses `( )`
 
-[Link title](links.md)
+## Link types
 
-```markdown
-[**Hi**, _I'm md_](links.md)
-```
+### Internal links
 
-[**Hi**, _I'm md_](links.md)
+Link between documentation files using either relative or absolute paths.
 
-## Anchor link
-
-You can link to a heading on a page with an anchor link. The link destination should be a `#` followed by the header text. Convert spaces to dashes (`-`).
+#### Relative paths
+Navigate relative to the current file's location:
 
 ```markdown
-I link to the [Inline link](#inline-link) heading above.
+[Security documentation](../security/index.md)
+
+[Monitoring guide](monitor/index.md)
 ```
 
-I link to the [Inline link](#inline-link) heading above.
+#### Absolute paths
+
+You can also use absolute paths to link to pages within the same repository:
 
 ```markdown
-I link to the [Notes](tables.md#notes) heading on the [Tables](tables.md) page.
+[API Keys](/deploy-manage/api-keys.md)
 ```
 
-## Cross Links
+### Same-page links (anchors)
 
-Cross links are links that point to a different docset.
+Link to sections within the same document using heading anchors prefixed with `#`:
 
 ```markdown
-[Cross link](kibana://cross-link.md)
+[Jump to the next section](#next-section-anchor)
 ```
 
-The syntax is `<scheme>://<path>`, where <scheme> is the repository name and <path> is the path to the file.
+Headings will automatically create anchor links in the resulting HTML. The are automatically generated by:
 
-## Auto Text Links
+- Converting to lowercase
+- Replacing spaces with hyphens
+- Removing special characters
 
-If you link to a local markdown file but omit any link text `docs-builder` will use the target's [title](titles.md).
+For example:
 
 ```markdown
-[](applies.md)
+### Same-page links (anchors)
+<!-- Creates anchor #same-page-links-anchors -->
 ```
-will output: [](applies.md)
 
-You can go one step further to auto generate link text for headings within files:
+#### Custom anchors
+You can specify custom anchors for headings inline:
 
 ```markdown
-[](applies.md#sections)
+## License management [#manage-license]
```

Comment:
Are you sure it requires `#`? (Or maybe it's just optional?)

Reply:
nice catch, fixed